Cargo.toml

```diff
@@ -576,11 +579,12 @@ disable-resolv-conf = []
 
 # grouping together features for benchmarks
 # excluing API client due to running out of memory during linking in Github Actions
-benches = ["sources", "transforms", "sinks", "vendor-all", "unix", "leveldb", "rdkafka-plain"]
-wasm-benches = ["transforms-add_fields", "transforms-field_filter", "transforms-wasm", "transforms-lua"]
-remap-benches = ["transforms-add_fields", "transforms-remap", "transforms-coercer", "transforms-json_parser"]
+benches-common = ["alloc-jemalloc"]
+benches = ["benches-common", "sources", "transforms", "sinks", "vendor-all", "unix", "leveldb", "rdkafka-plain"]
```

Comment:
I could also see renaming this to `benches-default` to match the suite name and then renaming `benches-common` to just `benches`.

Reply:
Done